configs/katello/4.0.yaml

```diff
@@ -5,6 +5,14 @@
   :4.0.0:
     :redmine_version_id: 1177
 :prior_releases:
+  :3.17.0:
+    :redmine_version_id: 1282
+  :3.17.1:
+    :redmine_version_id: 1337
+  :3.17.2:
+    :redmine_version_id: 1342
+  :3.17.3:
```

Comment:
Everything before 3.17.3 has gone GA by the time 4.0 branches so I don't think those are needed here? +1 to 3.17.3 though

Reply:
oops, you're right

Reply:
All fixed now.